PyInstaller/loader/rthooks/pyi_rth_qt5plugins.py

```diff
@@ -15,8 +15,11 @@
 import os
 import sys
 
-d = "qt5_plugins"
-d = os.path.join(sys._MEIPASS, d)
+
+sub_dir = os.path.join('PyQt5', 'Qt', 'plugins')
+d = os.path.join(sys._MEIPASS, sub_dir)
+if not os.path.isfile(d):
```

Comment:
>  it fails for onefile mode

Duh this was supposed to be `os.path.isdir(d)`, I bet that's the reason why it was failing for `onefile`.